sky/cli.py

```diff
@@ -3481,11 +3485,13 @@ def spot():
     default=None,
     is_flag=True,
     required=False,
-    help=('(Default: True; this flag is deprecated and will be removed in a '
-          'future release.) Whether to retry provisioning infinitely until the '
-          'cluster is up, if unavailability errors are encountered. This '
-          'applies to launching the spot clusters (both the initial and any '
-          'recovery attempts), not the spot controller.'))
+    help=(
+        '(Default: True; this flag is deprecated and will be removed in a '
+        'future release.) Whether to retry provisioning infinitely until the '
+        'cluster is up, if unavailability errors are encountered. This '
+        # pylint: disable=bad-docstring-quotes
```

Comment:
Reason we need to add these `pylint: disable=bad-docstring-quotes` inside and above `help`?

Reply:
Yea, I looked around but it seems to be a bug of the plugin... Not sure how to solve it. Let me add a comment then

Reply:
It seems `# noqa` does not work in this case, and I think  it should always be better to have the explict error instead of using a general marker?

Reply:
Adding
```
    # pylint: disable=bad-docstring-quotes
```
above the 3 occurrences of `help=...` seems to work.

Reply:
Ahh, good call. Moved. Thanks!

Reply:
Hmm, it does not work for me at L3492. Let me check...

Reply:
It seems we have to move it back. Could you show how do you place the L3492 to make it work?

Reply:
It seems for this help string, placing the disable hint on top of `help=...` doesn't work. This works for me and seems to read better:
```
    help=(
        '(Default: True; this flag is deprecated and will be removed in a '
        'future release.) Whether to retry provisioning infinitely until the '
        'cluster is up, if unavailability errors are encountered. This '
        'applies to launching the spot clusters (both the initial and any '  # pylint: disable=bad-docstring-quotes
        'recovery attempts), not the spot controller.'))
```

Reply:
Sounds good! Moved the comment. : )